dexcom/fetch/runner.go

```diff
@@ -363,8 +363,9 @@ func (t *TaskRunner) updateDeviceHash(device *dexcom.Device) bool {
 		t.deviceHashes = map[string]string{}
 	}
 
-	if device.SerialNumber != nil && t.deviceHashes[*device.SerialNumber] != deviceHash {
-		t.deviceHashes[*device.SerialNumber] = deviceHash
+	// TODO: SerialNumber is no longer sent, use transmitterId instead?
```

Comment:
seeking feedback on this

Reply:
I think there's not any other choice, assuming we actually get a `transmitterId`.